src/Core/Components/Button/FluentButton.razor.js

```diff
@@ -5,7 +5,12 @@ export function updateProxy(id) {
     }
     const element = document.getElementById(id);
 
-    if (!!element && !!element.form) {
-        element.proxy.setAttribute("form", element.form.id)
+    if (element.form) {
```

Comment:
Why did you removed `if (!!element && !!element.form)` ?
I think you need to check that `element` is not null... If it isn't, JS will throw an exception.

Reply:
In theory element can never by null, but adding check back to be on the safe(r) side